docs/xdr/FAQ.md

```diff
@@ -2,7 +2,9 @@
 
 **IP for `intake.sekoia.io` is `145.239.192.38`.**
 
-`intake.sekoia.io` is the domain name used to send your logs to SEKOIA.IO, either via Syslog or HTTP protocols. The IP address behind that service is static and stable. You can use that IP to configure your firewalls to allow connections from your forwarding systems to SEKOIA.IO.
+`intake.sekoia.io` is the domain name used to send your logs to SEKOIA.IO, either via Syslog or HTTP protocols. The IP address behind that service is static and stable. You can use that IP to configure your firewalls to allow connections from your forwarding systems to SEKOIA.IO. 
+
+Moreover, for all of our triggers available in our Playbooks section, used to retieve your logs, we use `141.95.157.224`. This IP will be useful to setup filtering if needed.
```

Comment:
IMHO this information should be moved to a dedicated section of this FAQ to avoid confusion (updated section is about “intake.sekoia.io”).